src/SIL.XForge.Scripture/ClientApp/src/app/checking/checking/checking.component.spec.ts

```diff
@@ -2179,7 +2179,11 @@ class TestEnvironment {
   }
 
   setTextFieldValue(textField: DebugElement, value: string): void {
-    const input = textField.query(By.css('input, textarea'));
+    // The text field may be either an input/textarea, or a component with an input/textarea within it.
```

Comment:
Can we remove MDC altogether? Seems like that'd be cleaner. I only see one remaining usage of it

Reply:
Plus I don't see a JIRA task that captures that last usage

Reply:
Talked in person. We'll put this work off until another subtask